apollo-compiler/src/test/graphql/com/example/input_object_type/ReviewInputExpected.java

```diff
@@ -0,0 +1,59 @@
+package com.example.input_object_type;
+
+import java.lang.Integer;
+import java.lang.String;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public final class ReviewInput {
```

Comment:
User should select composition over inheritance if he needs to add additional logic :)